src/main/java/life/qbic/io/commandline/PostmanCommandLineOptions.java

```diff
@@ -1,85 +1,161 @@
 package life.qbic.io.commandline;
 
-import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.List;
+import life.qbic.App;
+import life.qbic.ChecksumReporter;
+import life.qbic.FileSystemWriter;
+import life.qbic.io.parser.IdentifierParser;
+import life.qbic.model.download.Authentication;
+import life.qbic.model.download.QbicDataDownloader;
+import life.qbic.model.download.QbicDataStatus;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
 import picocli.CommandLine.Parameters;
 
-@Command(
-    name = "Postman",
-    footer =
-        "Optional: specify a config file by running postman with '@/path/to/config.txt'. Details can be found in the README.",
-    description =
-        "A client software for dataset downloads from QBiC's data management system openBIS.")
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+// main command with format specifiers for the usage help message
+@Command(name = "Postman",
+        footer = "Optional: specify a config file by running postman with '@/path/to/config.txt'. Details can be found in the README.",
+        description = "A client software for dataset downloads from QBiC's data management system openBIS.",
+        usageHelpAutoWidth = true,
+        sortOptions = false,
+        descriptionHeading = "%nDescription:%n",
+        parameterListHeading = "%nParameters:%n",
+        optionListHeading = "%nOptions:%n",
+        commandListHeading = "%nCommands:%n",
+        footerHeading = "%n")
+
 public class PostmanCommandLineOptions {
 
+  @Command(name = "download",
```

Comment:
I thought about that, especially because the download subcommand is quite big and also has its own options. I just wanted to make it work first and also have some doubts about putting the commands in separate classes, because they share a lot of options that have to be inherited and also because i was not sure if the @Command structure of picocli supports using separate classes. All examples I saw only had one class.